index.html

```diff
@@ -102,6 +102,7 @@ <h2 class="Formtitle">Contact me</h2>
             <form class="Contact-form" method="post" action="https://formspree.io/f/mrgjlvye" > 
                 <input class="inputclass" type="text" id="name" name="name"  required placeholder="Your name" maxlength="30"> 
                 <input class="inputclass" type="email" id="email" name="email" required placeholder="username@example.com" >
+                <span id="text"></span>
```

Comment:
- [ ] Good job working on the functionality, however, it is required that you show the error message to the user near the submit button informing them of the error and the form is not sent. Kindly place the message near the submit button.
![Capture](https://user-images.githubusercontent.com/85095702/162193365-074dec48-69e3-4f91-8746-032949f90863.PNG)